src/assets/scripts/client/changelog/ChangelogController.js

```diff
@@ -0,0 +1,200 @@
+import $ from 'jquery';
+import { SELECTORS } from '../constants/selectors';
+import { STORAGE_KEY } from '../constants/storageKeys';
+
+/**
+ * The controller class for the in-game changelog.
+ *
+ * @class ChangelogController
+ */
+export default class ChangelogController {
+    /**
+     * @constructor
+     * @param {ContentQueue} contentQueue
+     */
+    constructor(contentQueue) {
+        /**
+         * The content queue, used to load in the changelog data
+         *
+         * @property contentQueue
+         * @type {ContentQueue}
+         */
+        this.contentQueue = contentQueue;
+
+        /**
+         * The DOM of the changelog container, containing version, data, and dismiss.
+         *
+         * @property $changelogContainer
+         * @type {JQuery|HTMLElement}
+         */
+        this.$changelogContainer = $(SELECTORS.DOM_SELECTORS.CHANGELOG_CONTAINER);
```

Comment:
these should all be defaulted to `null` and then set within a `._createChildren()` method that's called before `._setupHandlers()`